### PR TITLE
Feature/json to csv

### PR DESCRIPTION
**Pull request by MarkRivera**
# Description
Fixes # (issue)

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [ ] Test A
- [ ] Test B

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by jstewart8053 — DISMISSED** [review]

**Comment by jstewart8053** [comment]
Looks good! You may want to clean up some of the unused variables so the checks pass.

**Review by mrockingham — DISMISSED** [review]
Looks good!

File: api/incidents/incidentsRouter.js [hunk 1]
```diff
@@ -291,4 +292,86 @@ router.post('/fetchfromds', async (req, res) => {
   }
 });
 
+router.get('/download', async (req, res) => {
+  try {
+    // Get Incidents from Database:
+    const incidents = await Incidents.getAllIncidents();
+    // Create Field Names:
+    const fields = [
```
Comment (jlong5795) [inline api/incidents/incidentsRouter.js]: This file is a bit long already. Let's try taking this array and putting it into a resource/utility type file and them importing it here instead.

**Review by jlong5795 — CHANGES_REQUESTED** [review]
This endpoint will allow a download of the entire database as a CSV which is great! Once we know this works we can do the same with a queried endpoint and allow a CSV download of a specific variety.

**Review by jlong5795 — APPROVED** [review]
Thank you for that revision. This will help keep this code maintainable.

**Pull request merged**